common/src/main/java/org/corfudb/common/metrics/micrometer/MeterRegistryProvider.java

```diff
@@ -41,6 +62,22 @@ public static synchronized void init(Logger logger, Duration loggingInterval,
             init(loggingInterval, localEndpoint, influxLineProtocolLoggingSink);
         }
 
+        /**
+         * Configure the meter registry of type LoggingMeterRegistry. All the metrics registered
+         * with this meter registry will be exported in the InfluxDB line protocol format
+         * (https://docs.influxdata.com/influxdb/v1.8/write_protocols/line_protocol_tutorial/)
+         * with  the provided loggingInterval frequency.
+         * @param logger A configured logger.
+         * @param loggingInterval A duration between log appends for every metric.
+         */
+        public static synchronized void init(Logger logger, Duration loggingInterval) {
```

Comment:
This `init` doesn't seem like its being used.

Reply:
Removed